src/hubverse_transform/model_output.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import pathlib
+import re
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+from pyarrow import csv, fs
+
+# Log to stdout
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s -  %(levelname)s - %(name)s - %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+class ModelOutputHandler:
+    def __init__(self, input_uri: str, output_uri: str):
+        input_filesystem = fs.FileSystem.from_uri(input_uri)
+        self.fs_input = input_filesystem[0]
+        self.input_file = input_filesystem[1]
+
+        output_filesystem = fs.FileSystem.from_uri(output_uri)
+        self.fs_output = output_filesystem[0]
+        self.output_path = output_filesystem[1]
+
+        # get file name and type from input file
+        path = pathlib.Path(self.input_file)
+        self.file_name = path.stem
+        self.file_type = path.suffix
+
+        # TODO: Add other input file types as needed
+        if self.file_type not in ['.csv', '.parquet']:
+            raise NotImplementedError(f'Unsupported file type: {path.suffix}')
+
+        # Parse model-output file name into individual parts
+        # (round_id, team, model)
+        file_parts = self.parse_file(self.file_name)
+        self.round_id = file_parts['round_id']
+        self.team = file_parts['team']
```

Comment:
This should be `'team_abbr'`

Reply:
Thanks, @annakrystalli...I pushed a fix for this: https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-transform/pull/1/commits/722165ac0208f260c3552112c35fc4f84c619422

(the fix is in a different part of the code, where we're specifying column names for the pyarrow table that gets written to the physical parquet file)